libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -407,7 +407,8 @@ void AC_PosControl::pos_to_rate_z()
 // calculates desired acceleration and calls accel throttle controller
 void AC_PosControl::rate_to_accel_z()
 {
-    const Vector3f& curr_vel = _ahrs.get_velocity();
+    Vector3f curr_vel;
```

Comment:
it was _inav.get_velocity() to should be get_velocity_NEU_cm(), it should merge the renaming with the changes ... the flow will be better